Kvantum/themes/kvthemes/KvArc/KvArc.kvconfig

```diff
@@ -230,10 +230,10 @@ indicator.element=tree
 
 [HeaderSection]
 inherits=PanelButtonCommand
-frame.top=2
-frame.bottom=2
-frame.left=2
-frame.right=2
+frame.top=0
+frame.bottom=1
+frame.left=0
+frame.right=1
 
```

Comment:
On 25/08/16 18:31, tsujan wrote:
>
> In Kvantum/themes/kvthemes/KvArc/KvArc.kvconfig 
> <https://github.com/tsujan/Kvantum/pull/97#discussion_r76288694>:
>
> > @@ -230,10 +230,10 @@ indicator.element=tree
> >
> >  [HeaderSection]
> >  inherits=PanelButtonCommand
> > -frame.top=2
> > -frame.bottom=2
> > -frame.left=2
> > -frame.right=2
> > +frame.top=0
> > +frame.bottom=1
> > +frame.left=0
> > +frame.right=1
> >
>
> This should be
>
> |frame.left=1 frame.right=0 |
>
> for header separators to be drawn because their width is that of left 
> frame.
>

Hmm... Don't want line on the left - as it then creates a double line 
with the itemview frame :-(